src/libcmd/installable-attr-path.cc

```diff
@@ -48,6 +48,26 @@ DerivedPathsWithInfo InstallableAttrPath::toDerivedPaths()
 {
     auto v = toValue(*state).first;
 
+    if (v->type() == nPath) {
+        auto storePath = v->path().fetchToStore(state->store);
+        return {{
+            .path = DerivedPath::Opaque {
+                .path = std::move(storePath),
+            },
+            .info = make_ref<ExtraPathInfo>(),
+        }};
+    }
+
+    else if (v->type() == nString) {
+        return {{
+            .path = state->coerceToDerivedPath(
+                noPos,
+                *v,
+                ""),
```

Comment:
> `""`

Shouldn't this include the attribute path?
We can't assume that it's obvious, because there could be multiple paths, and even a single path may not be apparent to the user when used in a script.

If the attr path is added elsewhere, then the equivalent for the flake attribute case might be redundant, unless it's only redundant when `--show-trace` is enabled.